src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerSearchControl.xaml.cs

```diff
@@ -23,6 +24,18 @@ public PackageManagerSearchControl()
         private void InitializeContext(object sender, RoutedEventArgs e)
         {
             PkgSearchVM = this.DataContext as PackageManagerSearchViewModel;
+
```

Comment:
Deter binding until after the ViewModel has been initialized in this view.

Reply:
This is because timing issue?